### PR TITLE
Deprecating NugetInstallerV0 and DownloadPackageV0 tasks

### PR DESCRIPTION
**Pull request by amp-powell**
**Task name**:  NuGetInstallerV0 and DownloadPackageV0

**Description**:  Added deprecated flag

**Documentation changes required:** After tasks are deprecated

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) User stories 1662927 and 2075334

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/NuGetInstallerV0/make.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
         {
             "items": [
                 "node_modules/azure-pipelines-tasks-packaging-common/node_modules/azure-pipelines-task-lib",
-                "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-task-lib"
+                "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-task-lib",
+                "node_modules/azure-pipelines-tool-lib/node_modules/azure-pipelines-task-lib"
```
Comment (zhenghao104) [inline Tasks/NuGetInstallerV0/make.json:7]: Why is this needed?

Comment (amp-powell) [reply Tasks/NuGetInstallerV0/make.json:7]: Some checks failed saying this appeared twice in node_modules, so I removed one of its instances

**Review by zhenghao104 — APPROVED** [review]

**Review by zhenghao104 — APPROVED** [review]
